webapp/src/Controller/Jury/JudgehostController.php

```diff
@@ -117,6 +119,44 @@ public function indexAction(Request $request): Response
                 'value' => 'j' . $lastJobId['jobid'],
             ];
 
+            $now = Utils::now();
+            $timings = $this->em->createQueryBuilder()
+                ->from(JudgeTask::class, 'jt')
+                ->join(JudgingRun::class, 'jr', 'WITH', 'jr.judgetask = jt')
+                ->join('jr.judging', 'j')
+                ->join('jt.judgehost', 'jh')
+                ->select('jr.endtime, jr.startTime')
+                ->andWhere('jt.judgehost = :judgehost')
+                ->andWhere('jr.startTime IS NOT NULL')
+                ->andWhere('jr.endtime IS NOT NULL')
```

Comment:
I guess this way we're ignoring currently running judging runs. Could we also include those?